pytorch_blade/tests/mhlo/matmul.mlir

```diff
@@ -20,42 +21,44 @@ func.func @torch.aten.bmm(%arg0: !torch.vtensor<[10,3,4],f32>, %arg1: !torch.vte
 }
 
 // -----
-// CHECK-LABEL:  func @torch.aten.bmm.dyn(
+
+// CHECK-LABEL:  func.func @torch.aten.bmm.dyn(
 // CHECK-SAME:         %[[ARG0:.*]]: tensor<4x?x256xf32>, %[[ARG1:.*]]: tensor<4x256x?xf32>) -> tensor<4x?x?xf32> {
-// CHECK:         %[[T0:.*]] = "mhlo.dot_general"(%[[ARG0]], %[[ARG1]]) {dot_dimension_numbers = #mhlo.dot<lhs_batching_dimensions = [0], rhs_batching_dimensions = [0], lhs_contracting_dimensions = [2], rhs_contracting_dimensions = [1]>} : (tensor<4x?x256xf32>, tensor<4x256x?xf32>) -> tensor<4x?x?xf32>
-// CHECK:         return %[[T0]] : tensor<4x?x?xf32>
+// CHECK:         %[[C256_I32:.*]] = arith.constant 256 : i32
+// CHECK:         %[[C4_I32:.*]] = arith.constant 4 : i32
```

Comment:
maybe don't have to check the constants if not critical? I saw unstable order of in sequence of constant IR.

Reply:
TorchMLIR checks all the lines of output IR.